commands/meshuggah.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = function meshuggah (controller) {
+  // Serve up a Meshuggah video
+  controller.hears('Meshuggah', params.triggerEvents, function(bot,message) {
```

Comment:
`params` is not defined. You can simply omit this as controller.hears only needs 2 parameters.